src/components/Facility/ConsultationDetails/QuestionnaireResponsesList.tsx

```diff
@@ -124,17 +125,20 @@ function QuestionGroup({
   );
 }
 
-export default function QuestionnaireResponsesList({ encounter }: Props) {
+export default function QuestionnaireResponsesList({
+  encounter,
+  patientId,
+}: Props) {
   const { t } = useTranslation();
 
   return (
     <PaginatedList
       route={routes.getQuestionnaireResponses}
       pathParams={{
-        patientId: encounter.patient.id,
+        patientId: patientId,
       }}
       query={{
-        encounter: encounter.id,
+        ...(encounter && { encounter: encounter.id }),
```

Comment:
isn't this more readable? or am i missing something?

```suggestion
        encounter: encounter?.id,
```